### ISSUE TITLE
Future Website - Contact Us Page

### ISSUE THREAD

**Issue by ronenreinheimer** [206Tours/DB#354]
Hi Brad,

Sandy would like you to make this form work:
http://www.206tours.com/contact/index3-test2.htm

we are going to go live May 1, if you can work on this soon, it would be great. You can give Sandy / Milanka a quote for this project.

I setup the scroll element as a background image, so maybe you can create the form on top of it and line it up? let me know how i can help.

feel free to modifiy the .htm file.

Thank you!

**Comment by bradatkins** [comment]
It's an image? There is no form?

**Comment by ronenreinheimer** [comment]
That is correct, it is only a background image. We can build a form over it. can you do it or you need my help?

God Bless,

________________________________
[http://www.206tours.com/sigs/30th-logo2.png]<http://www.206tours.com/>

Ronen Reinheimer
Art Director
206Tours.com<http://www.206tours.com>

**Comment by ronenreinheimer** [comment]
I suggest you build the form over it and I'll take it from there

**Comment by ronenreinheimer** [comment]
OK! Can you make the form transparent? So it will seem like they are entering the information into the image? Does that make sense?

God Bless,

________________________________
[http://www.206tours.com/sigs/30th-logo2.png]<http://www.206tours.com/>

Ronen Reinheimer
Art Director
206Tours.com<http://www.206tours.com>

**Comment by ronenreinheimer** [comment]
Hi Brad,

What do you think? Is this enough for you to work with? You know more about this than I.
http://www.206tours.com/contact/index3-test2.htm


God Bless,

________________________________
[http://www.206tours.com/sigs/30th-logo2.png]<http://www.206tours.com/>

Ronen Reinheimer
Art Director
206Tours.com<http://www.206tours.com>

**Comment by ronenreinheimer** [comment]
I'll give it a try. No promises :)

**Comment by ronenreinheimer** [comment]
Hi Brad,

We both know you have the skills!

God Bless,

________________________________
[http://www.206tours.com/sigs/30th-logo2.png]<http://www.206tours.com/>

Ronen Reinheimer
Art Director
206Tours.com<http://www.206tours.com>
